### DISCUSSION TITLE
How can I show Popover content as html?

### DISCUSSION THREAD

**Discussion by ptmcode** [tabler/tabler#1978] [General]
I can see the popover document with plain text but not html.

https://tabler.io/docs/components/popover

**Comment by ethancrawford** [comment]
@ptmcode - in case you're still wondering, it's described in Bootstrap's [popover documentation](https://getbootstrap.com/docs/5.3/components/popovers/) (it's disabled by default and Tabler leaves that unchanged). You'd need to probably enable it in one of two ways: 
- add `data-bs-html="true"` to the popover's triggering element along with the html added into the content, or
- add a template as described in the [popover options](https://getbootstrap.com/docs/5.3/components/popovers/#options) and use `setContent(...)` as described in the [popover methods](https://getbootstrap.com/docs/5.3/components/popovers/#methods) to inject plain text into the custom html content.
(Since it allows content passed as an object, as quoted on that page):
> The setContent method accepts an object argument, where each property-key is a valid string selector within the popover template, and each related property-value can be string | element | function | null

(Adding html via `data-bs-html="true"` is of course risky unless you know you have complete control over the html that would end up in the popover, rather than possibly having user specified content end up there).
